.github/workflows/int-and-connected-test-run.yml

```diff
@@ -78,51 +78,58 @@ jobs:
       - name: "Checkout code"
         uses: actions/checkout@v2
       - name: "Whitelist Runner IP"
-        uses: broadinstitute/datarepo-actions/actions/main@0.57.0
+        uses: broadinstitute/datarepo-actions/actions/main@0.59.0
         with:
           actions_subcommand: 'gcp_whitelist'
           role_id: ${{ secrets.ROLE_ID }}
           secret_id: ${{ secrets.SECRET_ID }}
       - name: "Check for an available namespace to deploy API to and set state lock"
-        uses: broadinstitute/datarepo-actions/actions/main@0.57.0
+        uses: broadinstitute/datarepo-actions/actions/main@0.59.0
         with:
           actions_subcommand: 'k8_checknamespace'
           k8_namespaces: 'integration-1,integration-2,integration-3,integration-6'
       - name: "Build docker container via Gradle"
-        uses: broadinstitute/datarepo-actions/actions/main@0.57.0
+        uses: broadinstitute/datarepo-actions/actions/main@0.59.0
         with:
           actions_subcommand: 'gradlebuild'
       - name: "Deploy to cluster with Helm"
-        uses: broadinstitute/datarepo-actions/actions/main@0.57.0
+        uses: broadinstitute/datarepo-actions/actions/main@0.59.0
         with:
           actions_subcommand: 'helmdeploy'
           helm_create_secret_manager_secret_version: '0.0.6'
           helm_datarepo_api_chart_version: 0.0.71
           helm_datarepo_ui_chart_version: 0.0.61
           helm_gcloud_sqlproxy_chart_version: 0.19.7
           helm_oidc_proxy_chart_version: 0.0.22
+      - name: "Fetch gitHash for deployed integration version"
+        id: configuration
+        run: |
+          git_hash=$(git rev-parse --short HEAD)
+          echo "::set-output name=git_hash::$git_hash"
+          echo "Latest git hash for this branch: $git_hash"
```

Comment:
@fboulnois - I know you're not a fan of outputs. Do you think it would be better to just go ahead and set the env variable here?

Reply:
I go back and forth between choices. This seems fine.